GameServerLib/Content/ItemCollection.cs

```diff
@@ -58,5 +66,31 @@ public static ItemContentCollection LoadItemsFrom(string directoryPath)
 
             return result;
         }
+
+        public static ItemContentCollection LoadItemsFromZip(string zipLocation, string packageName)
```

Comment:
Please use the abstraction of `IPackage` to avoid different functions for different packages. There should be one function like `LoadItemsFromPackage(IPackage)` that handles all the cases. (zip, etc)

Moreover, if this is static function it should moved to the `ZipPackage`/`PackageBase` function without getting any parameters. (as it uses the `this` itself)